src/sail_json_backend/json.ml

```diff
@@ -364,6 +364,14 @@ let parse_type_union i ucl =
   match ucl with
   | Tu_aux (Tu_ty_id (c, d), annot) ->
       debug_print ~printer:prerr_string ("Tu_ty_id " ^ string_of_id d ^ "(");
+      List.iter
+        (fun attr ->
+          match attr with _, "name", Some (AD_aux (AD_string s, _)) -> Hashtbl.add names (string_of_id d) s | _ -> ()
```

Comment:
That's a long line. Is that your choice of formatting or did `dune fmt` suggest it? If it was `dune fmt`, so be it. If we have the option to change it, though, I prefer if each match case was on it's own line:
```
match attr with
  _, "name" [...]
  _ -> ()
```

Reply:
It's as a result of  `dune fmt `, for both name and description blocks